app/services/sends_assignment_reminders.rb

```diff
@@ -1,11 +1,11 @@
 class SendsAssignmentReminders
-  def initialize
-    @week = Week.upcoming.first
-  end
-
   def call
-    Assignment.where(week: @week).find_each do |assignment|
-      UserMailer.assignment_reminder(assignment).deliver
+    Office.find_each do |office|
+      @week = office.weeks.upcoming.second
```

Comment:
second? Is the actual problem in `upcoming`?

Reply:
upcoming includes the week it currently is. I could change upcoming but that would mean having to change a lot of other things (the schedule uses it to display only the assignments from the current week on).

Reply:
Assignment model also uses `upcoming` and it makes some sense there to include the current week. Yeah, this seems fine.